src/data/rules.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "rule": "1. Formação de equipas",
```

Comment:
Minor thing: if the rules are going to have a number, you could consider using the index of the element instead of hard coding it in the text.